api/db.json

```diff
@@ -0,0 +1 @@
+{"uploads":[{"name":"IMG_0599.JPG","type":"image/jpeg","size":343845,"path":"/tmp/uploads/upload_7b8d12d21774c726904d6272f6176d57"},{"name":"IMG_0599.JPG","type":"image/jpeg","size":343845,"path":"/tmp/uploads/upload_e39a69250284e0113b276e3dcf87dbb4"},{"name":"kalen-emsley-99660.jpg","type":"image/jpeg","size":14036175,"path":"/tmp/uploads/upload_fe22084971ed01e6849cd7fa253ccf4b"}]}
```

Comment:
Need to add this file to the gitignore, and remove what was already committed here.